odra-macros/src/ast/host_ref_item.rs

```diff
@@ -198,7 +198,7 @@ mod ref_item_tests {
                     self.env.last_call().contract_last_call(self.address)
                 }
 
-                pub fn try_total_supply(&self) -> Result<U256, OdraError> {
+                pub fn try_total_supply(&self) -> Result<U256, odra::OdraError> {
```

Comment:
I just noticed this. Do we need `try_` for `&self` methods?

Reply:
Yes. First, for consistency. Second, a `&self` function may have guards and fail for some reason. But testing your code you can use a regular function if you are sure it does not fail.